src/app/Caches/DetailCache.tsx

```diff
@@ -109,6 +144,29 @@ const DetailCache = (props: { cacheName: string }) => {
     );
   };
 
+  const encodingMessage = () => {
```

Comment:
@karesti This function is not used anywhere, is it anything we need in the future?

Reply:
forgotten by mistake in the pr